pages/games/[gameId].tsx

```diff
@@ -205,8 +214,8 @@ function GamePage(): JSX.Element {
                 >
                   {shortenAddress(id)}
                 </Link>
-                {isMaster && (
-                  <Button onClick={updateGameMetadata.onOpen} size="sm">
+                {(isAdmin || isMaster) && (
```

Comment:
or isOwner ?

Reply:
Add in bc64c93db8a8398f5e6942b507d76b90e7a973f6